tutorials/angular/start/stocks/src/app/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface Stock {
+    Ask: number;
+    Bid: number;
+    BPOD: string;
+    Bloomberg: string;
+    Description: string;
+    Exchange: string;
+    RIC: string;
+    Venues: string;
+}
+

```

Comment:
Also include Client interface:
```suggestion
export interface Client {
  accountManager: string;
  address: string;
  branchCode: string;
  contactNumbers: string;
  eId: string;
  email: string;
  gId: string;
  id: string;
  name: string;
  pId: string;
  portf: string;
  portfolio: string;
  preferredName: string;
  spn: string;
  notes: string[];
}
```

Reply:
applied